pkg/cmd/roachtest/monitor.go

```diff
@@ -17,10 +17,66 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/roachprod"
 	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
 	"github.com/cockroachdb/cockroach/pkg/roachprod/logger"
+	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/errors"
 	"golang.org/x/sync/errgroup"
 )
 
+// monitorProcess represents a single process that the monitor monitors.
+type monitorProcess struct {
+	node               install.Node
+	virtualClusterName string
+	sqlInstance        int
+}
+
+// MonitorExpectedProcessHealth represents the expected health of a process.
+type MonitorExpectedProcessHealth string
+
+const (
+	ExpectedHealthy = MonitorExpectedProcessHealth("process healthy")
```

Comment:
Nit: maybe `ExpectDead` and `ExpectAlive` to make english work better? Otherwise, we're mixing adjectives and nouns.

Reply:
Yeah I like it, also removes the confusion between `healthy` being a state of `health`